app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -40,6 +40,60 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
   const { inPersonFullAddressEntryEnabled, inPersonURL, skipDocAuth, skipDocAuthFromHandoff } =
     useContext(InPersonContext);
   const appName = getConfigValue('appName');
+  const inPersonLocationPostOfficeSearchForm = inPersonFullAddressEntryEnabled
+    ? InPersonLocationFullAddressEntryPostOfficeSearchStep
+    : InPersonLocationPostOfficeSearchStep;
+
+  // Define different states to be used in human readable array declaration
+  const documentFormStep = {
+    name: 'documents',
+    form: DocumentsStep,
+    title: t('doc_auth.headings.document_capture'),
+  } as FormStep;
+  const reviewFormStep = {
+    name: 'review',
+    form:
+      submissionError instanceof UploadFormEntriesError
+        ? withProps({
+            remainingSubmitAttempts: submissionError.remainingSubmitAttempts,
+            isResultCodeInvalid: submissionError.isResultCodeInvalid,
+            isFailedResult: submissionError.isFailedResult,
+            isFailedDocType: submissionError.isFailedDocType,
+            isFailedSelfie: submissionError.isFailedSelfie,
+            isFailedSelfieLivenessOrQuality:
+              submissionError.selfieNotLive || submissionError.selfieNotGoodQuality,
+            captureHints: submissionError.hints,
+            pii: submissionError.pii,
+            failedImageFingerprints: submissionError.failed_image_fingerprints,
+          })(ReviewIssuesStep)
+        : ReviewIssuesStep,
+    title: t('errors.doc_auth.rate_limited_heading'),
+  } as FormStep;
+
+  // In Person Steps
+  const prepareFormStep = {
+    name: 'prepare',
+    form: InPersonPrepareStep,
+    title: t('in_person_proofing.headings.prepare'),
+  } as FormStep;
+  const locationFormStep = {
+    name: 'location',
+    form: inPersonLocationPostOfficeSearchForm,
+    title: t('in_person_proofing.headings.po_search.location'),
+  } as FormStep;
+  const hybridFormStep = {
+    name: 'switch_back',
+    form: InPersonSwitchBackStep,
+    title: t('in_person_proofing.headings.switch_back'),
+  } as FormStep;
```

Comment:
I'd recommend using type assignment for this rather than casting, since casting can have the tendency to hide some real issues.

```suggestion
  const prepareFormStep: FormStep = {
    name: 'prepare',
    form: InPersonPrepareStep,
    title: t('in_person_proofing.headings.prepare'),
  };
  const locationFormStep: FormStep = {
    name: 'location',
    form: inPersonLocationPostOfficeSearchForm,
    title: t('in_person_proofing.headings.po_search.location'),
  };
  const hybridFormStep: FormStep = {
    name: 'switch_back',
    form: InPersonSwitchBackStep,
    title: t('in_person_proofing.headings.switch_back'),
  };
```